### PR TITLE
frontend: added outputs

### PR DESCRIPTION
**Pull request by adamleithp-mystic**
# Pull request outline

## Checklist:
- [ ] Docs updated
- [ ] Tests written
- [ ] Check for breaking changes in Resource
- [ ] Version bumped

Added:
- A list of new things

Changed:
- A list of things changed

Removed:
- A list of things removed

## Related issues:
_none_

___

- _You can add a reference to an issue using hash followed by the issue number_
- _If a checklist item is to be ignored it must be struck through_

### CODE REVIEW

File: pipeline/container/frontend/index.html [hunk 1]
```diff
@@ -692,22 +919,10 @@
...
-              setDynamicFields(
-                generateDynamicFieldsFromIOVariables(data.input_variables)
-              );
-            })
-            .catch((err) => console.log(err));
-        }, []);
-        console.log(pipeline);
-        React.useEffect(() => {
-          setInputData(generateInputDataArrayFromDynamicFields(dynamicFields));
-        }, [dynamicFields]);
```
Comment (plutopulp) [inline pipeline/container/frontend/index.html:710]: What handles the inputData updates now then? (When the form values change)

Comment (adamleithp-mystic) [reply pipeline/container/frontend/index.html:710]: you're right!

Comment (adamleithp-mystic) [reply pipeline/container/frontend/index.html:710]: re-added! good catch

File: pipeline/container/frontend/index.html [hunk 2]
```diff
@@ -759,57 +972,265 @@
...
+                data.input_variables
+              );
+              // const dynamicFields =
+              //   generateDynamicFieldsFromIOVariables(mockInputIOVariables);
+
+              setDynamicFields(dynamicFields);
+
+              const inputData =
+                generateInputDataArrayFromDynamicFields(dynamicFields);
+              setInputData(inputData);
```
Comment (plutopulp) [inline pipeline/container/frontend/index.html:1168]: I guess this is not needed anymore due the useEffect, e.g. inputData will be set twice on mount. But no biggie if you want to leave it and merge

**Pull request merged**